apps/app/repository.tf

```diff
@@ -12,4 +12,162 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-// TODO: GH stuff will go here
+resource "github_repository" "app" {
+  name        = var.app_id
```

Comment:
I guess this is obvious in TRE land but `app_id` feels like it should be a uid or a azure resource id, to me at least. I'd prefer `app_name`

Reply:
It should be a uid as it needs to be unique - there's another field for a friendly app name

Reply:
Or are you saying just for this resource we should use the app name instead? There's a danger of conflicts if two apps have the same friendly name

Reply:
yeah sorry meant like a guid. ignore me. think it's clear